index.html

```diff
@@ -551,6 +551,68 @@ <h2>Exposing MediaStreamTrack source background blur support</h2>
 };</pre>
     </div>
   </section>
+  <section>
+    <h2>Exposing MediaStreamTrack source eye gaze correction support</h2>
+    <p>Some platforms or User Agents may provide built-in support for human eye
+       gaze correction, in particular for camera video streams.
+       This may for instance allow the web application to update its UI or to
+       not apply human eye gaze correction on its own.
+       For that reason, we extend {{MediaStreamTrack}} with the following
+       properties.
+    </p>
+    <p>The WebIDL changes are the following:</p>
+    <pre class="idl"
+>partial dictionary MediaTrackSupportedConstraints {
+  boolean eyeGazeCorrection = true;
+};
+
+partial dictionary MediaTrackCapabilities {
+  sequence&lt;boolean&gt; eyeGazeCorrection;
+};
+
+partial dictionary MediaTrackConstraintSet {
+  ConstrainBoolean eyeGazeCorrection;
+};
+
+partial dictionary MediaTrackSettings {
+  boolean eyeGazeCorrection;
+};</pre>
+    <section>
+      <h3>Processing considerations</h3>
+      <p>When the "eyeGazeCorrection" setting is set to <code>true</code> by
+         the <a>ApplyConstraints algorithm</a>, the User Agent will attempt to correct
+         human eye gaze.
+      </p>
+      <p>When the "eyeGazeCorrection" setting is set to <code>false</code> by
+         the <a>ApplyConstraints algorithm</a>, the User Agent will not correct human
+         eye gaze.
+      </p>
+    </section>
+    <section>
+      <h3>Examples</h3>
+      <pre class="example">
+&lt;video&gt;&lt;/video&gt;
+&lt;script&gt;
+// Open camera.
+const stream = await navigator.mediaDevices.getUserMedia({video: true});
+const [videoTrack] = stream.getVideoTracks();
+
+// Try to correct eye gaze.
+const videoCapabilities = videoTrack.getCapabilities();
+if ((videoCapabilities.eyeGazeCorrection || []).includes(true)) {
+  await videoTrack.applyConstraints({eyeGazeCorrection: {exact: true}});
+} else {
+  // Eye gaze correction is not supported by the platform or by the camera.
+  // Consider falling back to some other method.
+}
+
```

Comment:
```suggestion
const capabilities = videoTrack.getCapabilities();
if (("eyeGazeCorrection" in capabilities) {
  await videoTrack.applyConstraints({eyeGazeCorrection: true});
} else {
  // Eye gaze correction is not supported by the platform or by the camera.
  // Consider falling back to some other method.
}
```

Reply:
That's not the same thing.

`if ((videoCapabilities.eyeGazeCorrection || []).includes(true))` tests if eye gaze correction can be enabled and the else branch can then fall back to some other method.

`if ("eyeGazeCorrection" in capabilities)` tests only if the UA and the track know about eye gaze correction but the capability can be either `[false]`, `[false, true]` or `[true]`. The `await videoTrack.applyConstraints({eyeGazeCorrection: true});` will of course always succeed regardless the capability of the lack of it because only optional constraints are used. But now the else branch is reached only if the is no eye gaze correction capability but not if the eye gaze correction capability is `[false]` which contradicts the comment in that else branch.